source/js/buyers-guide/bg-main.js

```diff
@@ -19,6 +19,7 @@ import initializeSentry from "../common/sentry-config.js";
 import PNIMobileNav from "./pni-mobile-nav.js";
 
 // Initializing component a11y browser console logging
+// TODO React-axe is currently deprecated, we should replace it with @axe-core/react
```

Comment:
Can you create a ticket for this and update this inline comment to include link to that ticket, please?

Reply:
https://github.com/MozillaFoundation/foundation.mozilla.org/issues/10306